modules/indexing/src/main/java/org/apache/ignite/internal/processors/query/h2/IgniteH2Indexing.java

```diff
@@ -910,7 +930,10 @@ public ResultSet executeSqlQueryWithTimer(
 
         enableDataPageScan(dataPageScanEnabled);
 
-        try {
+        try (TraceSurroundings ignored = MTC.support(ctx.tracing()
+            .create(SQL_QRY_EXECUTE, MTC.span())
+            .addTag(SQL_QRY_TEXT, () -> sql))
```

Comment:
That might include sensitive information. Are we safe?

Reply:
This tag will not include data if a parameterized query is used. If we skip this, it’s rather difficult to determine which query the current span belongs to.

Reply:
I agree that in case of skipping such data it'll be difficult to determine which query the current span belongs to. However even parameterized query is used. Do we use includeSencitive logic/flag in case of converting sql to string?

Reply:
No. We don't. Maybe it worth then to add includeSensitive flag in tracing configuration. WDYT?